Using Neptune with SageMaker Training Jobs example (#166)
neptune-ai/examples: how-to-guides/sagemaker/README.md

> # Using Neptune client with AWS SageMaker

Reviewer: ```suggestion
# Using Neptune client with Amazon SageMaker
```

> The following examples show how to use Neptune client with AWS SageMaker. The examples use Jupyter notebooks. The

Reviewer: ```suggestion
The following examples show how to use the Neptune client with SageMaker. The examples use Jupyter notebooks. The
```

> notebooks in most cases should be run from AWS SageMaker notebooks or AWS SageMaker Studio notebooks, the information

Reviewer: ```suggestion
notebooks in most cases should be run from either SageMaker notebooks or SageMaker Studio notebooks --
```

> regarding how they should be used is provided within the notebooks.

Reviewer: ```suggestion
you'll find this information within the notebook itself.
```

For running the example of using the custom docker container, you will additionally need to install [docker] and
[AWS CLI] tools.

 [docker]: https://docs.docker.com/engine/install/

>  [AWS CLI]: https://aws.amazon.com/cli/

Reviewer 2: I meant updating the [repository readme](https://github.com/neptune-ai/examples#readme) :)

Author: Done.
